internal/providers/docker/compose/yaml.go

```diff
@@ -52,7 +52,9 @@ type Service struct {
 	// TODO: cap_add
 	// TODO: cap_drop
 	// TODO: cgroup_parent
-	Command       Command  `yaml:"command"`
+
+	Command Command `yaml:"command"`
+
```

Comment:
what's with the extra whitespace here?

Reply:
The go commenting convention is to have a comment directly preceding the thing you're documenting. This means that tools expect any comment directly above a struct member to be documentation for that struct member. So the documentation for `Command`, as reported by `go doc` and IDEs/language servers, was "TODO: cap_add TODO: cap_drop".

But the whitespace below this line was unnecessary. Will remove.